tt_metal/hw/inc/dataflow_api.h

```diff
@@ -321,12 +329,22 @@ constexpr inline std::int32_t get_tile_size(const std::int32_t operand) {
     std::uint32_t input = operand;
 
     // L1 16B words
-    std::uint32_t num_words = GET_TILE_SIZE((uint)unpack_src_format[input]);
+    std::uint32_t num_words = (uint)unpack_tile_size[input];
```

Comment:
nit : cast to the same type, although it should be okay on all architectures we use.